sdk/storage/azure-storage-blob/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.2.1 (Unreleased)
 
+**Breaking changes**
+- The `StorageUserAgentPolicy` is now replaced with the `UserAgentPolicy` from azure-core.
```

Comment:
It's not a breaking change from a customer point of view, since the UA string is generated by our code and interpreted by our internal private telemetry at microsoft.
At best I would do a "note" section, since it's not really a bug fixes neither.